aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -87,6 +87,12 @@
    */
   public static final String GLUE_CATALOG_ID = "glue.id";
 
+  /**
+   * Used by {@link LakeFormationAwsClientFactory}.
```

Comment:
nit: no need to say it's used by the client factory, it can be used in the future with other features like Glue resource link.